text/0033-input-versions.md

```diff
@@ -0,0 +1,166 @@
+# Input Version Annotation
+
+## Related issues and PRs
+
+- Reference Issues:
+- Implementation PR(s):
+
+## Timeline
+
+- Started: 2013-10-13
+
+## Summary
+
+Reserve a way to indicate the Cedar version of an input format, so that future versions of Cedar can recognize and gracefully handle earlier input formats.
```

Comment:
love it, lets us bump versions (including to those with breaking changes) without needing to refactor our codebase immediately

Reply:
We need to be clear about "gracefully handle earlier input formats". It can be as simple as returning an error message saying you are using an old version of the schema and need to update.